CedarJava/src/main/java/com/cedarpolicy/model/policy/TemplateLink.java

```diff
@@ -21,32 +21,33 @@
 import java.util.List;
 import com.google.common.collect.ImmutableList;
 
-/** Template instantiation. */
-public class TemplateInstantiation {
+/** Template-linked policy. */
+public class TemplateLink {
 
     @JsonProperty("templateId")
     private final String templateId;
 
     @JsonProperty("resultPolicyId")
     private final String resultPolicyId;
 
-    private final List<Instantiation> instantiations;
+    @JsonProperty("instantiations")
```

Comment:
I'm guessing the cedar-policy ffi interface relies on this Json property name and will be renamed in a later change?

Reply:
Yep, coming soon.